src/main/java/pages/TestReviewPage.java

```diff
@@ -37,7 +37,14 @@ public void clickChangeDetailsButton() {
     }
 
     public boolean isChangeDetailsButtonDisplayed() {
-        return findElementById(CHANGE_DETAILS_BUTTON_ID).isDisplayed();
+        boolean status;
```

Comment:
Why did you modify the method? And why did you took this approach?

Reply:
The element does not exist in the DOM and the initial method was throwing exceptions instead of False.

Reply:
More specific exception handling not Exception e

Reply:
done